mods/d2k/rules/aircraft.yaml

```diff
@@ -72,16 +49,17 @@ frigate:
 		Name: Frigate
 		Description: Supply spacecraft
 	Plane:
-		ROT: 5
-		Speed: 350
-		RepairBuildings: repair
+		CruiseAltitude: 1800
+		ROT: 7
+		Speed: 296	##148
```

Comment:
Actually this is still a lot slower than the original at max speed, which is how I hear people play it. But I do agree that trikes are unkillable while moving.
I tried the commented-out values though, and it is indeed ridiculously slow.